pucauto.py

```diff
@@ -10,10 +10,21 @@
 from bs4 import BeautifulSoup
 from lib import logger
 
-
-with open("config.json") as config:
-    CONFIG = json.load(config)
-
+try:
```

Comment:
I'd prefer to remove this check and just let it blow up if they don't have a config.py. Less code. They'll still get an error mentioning config.py.